sig-cli/CONTRIBUTING.md

```diff
@@ -66,20 +73,7 @@ Since maintainers will need to own any code for these contributions, these shoul
 be very limited in scope and contain minimal risk
 (e.g. simple regression fixes, improved documentation, improved testing).
 
-## Before You Begin
-
-**Note**: Complete the following steps before reaching out to cli community members.
-
-### Agree to contribution rules
-
-Follow the [CLA signup instructions](../CLA.md).
-
-### Learn a bit about the kubectl cli
-
-This is important.
-
-- Learn about using kubectl with Kubernetes in the [Kubernetes Basics Tutorial].
-- Learn about managing configuration in the [kubectl docs].
+- ​
```

Comment:
Remove the bullet point.

Reply:
The required changes have been incorporated. @idvoretskyi, you can proceed with your review.

Reply:
@nidorano, please remove the bullet point on line #73, which in unnecessary (it seems, it left there after the page reformatting).

Otherwise LGTM

Reply:
@idvoretskyi, the bullet point was removed.